pxl_scripts/px/services_to_ips/services_to_ips.pxl

```diff
@@ -0,0 +1,36 @@
+# Finds services that have traffic to and from a list of IP addresses.
+# Calculates bandwidth in bytes per second (BPS).
+
+import px
+
+window_size_seconds = 10
+
+df = px.DataFrame('conn_stats')
+
+# Add IPs of the remote endpoint to filter.
+expr = df.remote_addr == '10.4.1.1'
+expr = expr or df.remote_addr == '10.4.1.2'
```

Comment:
Can we make this and argument to the script?

Reply:
We do not support lists as args at the moment, will want to hardcode for now.